### PR TITLE
Updated version

### PR DESCRIPTION
**Pull request by JackGraymer**
Mostly adds CSS, framework and some custom classes.

Modifies html structure to fit styles.

Changes javascript enough to change the creation of HTML elements, add theme button

### CODE REVIEW

**Comment by JackGraymer** [comment]
> Is it a linter that changed the style of the JavaScript? I'm okay with
that, I just don't want future development to cause back and forth.
- I think so, linter or prettier could have done it, that part of the code
was not touched by me.

>Please remove space before "Next"
- Done

>Could you use HTML5 LocalStorage to save whether the user has selected the
light theme, so if they go back later, it is the light theme?
- I will look into it

**Comment by JackGraymer** [comment]
Fixed errors, linting and prettier are the cause for the syntax differences.

Added theme feature to local Storage

File: README.md [hunk 1]
```diff
@@ -1,23 +1,65 @@
...
 And the following keys are optional:
- - required:             Extra validation in the form of case-insensitive required strings
- - custom_error_message: If the validation fails, show this error message to the user
+
+- required: Extra validation in the form of case-insensitive required strings
+- custom_error_message: If the validation fails, show this error message to the user
+
+## Re-design
+
+I came across this site as part of an open source Full Stack Web Development course, [The Odin Project](https://www.theodinproject.com/dashboard).
```
Comment (rhc2104) [inline README.md:30]: This should be written in the third person.

Alvaro Cervan came across this site as part of an open source Full Stack Web Development course, [The Odin Project](https://www.theodinproject.com/dashboard).

File: README.md [hunk 2]
```diff
@@ -1,23 +1,65 @@
...
+
+I loved this SQL interactive tutorial; it is by far the best resource that I have found. It is totally interactive and provides query examples with live results.
```
Comment (rhc2104) [inline README.md:32]: He loved this SQL interactive tutorial; it is by far the best resource that he has found. It is totally interactive and provides query examples with live results.

File: README.md [hunk 3]
```diff
@@ -1,23 +1,65 @@
...
+
+However, the UI looked **very** dated. In fact, when checking the commits, some are from 9 years ago, which explains this issue.
+
+I then started a project to update the UI, redesign its structure, and add support for smaller devices.
```
Comment (rhc2104) [inline README.md:36]: He then started a project to update the UI, redesign its structure, and improve support for smaller devices.

**Comment by JackGraymer** [comment]
readme updated from 1st person to 3rd

**Pull request merged**